.github/workflows/wheels.yml

```diff
@@ -10,7 +10,7 @@ on:
     types: [published]
 
 env:
-  CIBW_ENVIRONMENT: "PIP_ONLY_BINARY=:all:"
+  CIBW_BEFORE_BUILD: "python -m pip install numpy --only-binary=:all:"
```

Comment:
Why did you revert this? You don't need NumPy to build since you have built on pybind11, so you either need `CIBW_BEFORE_TEST` or better yet, `CIBW_ENVIRONMENT`.

Reply:
Perhaps you are right, perhaps you are wrong, but setting this up took a lot of time and I am not interested in touching this.

Reply:
Unless something is broken, please don't "fix" it in my repositories.

Reply:
Thanks for the vote of confidence.

Reply:
I have not touched anything in `github.com:hdembinski`. I am responsible for maintaining the build systems for Scikit-HEP, and am trying to work on enabling our packages to be used in as many places as possible, such as on PyPy, ARM, and Apple Silicon. If you tie NumPy to the build step, then you will not be able to support builds where NumPy is not available as a wheel without building it yourself (which is really slow on ARM/PyPy, and hard on Apple).

As lead maintainer, it is your decision to accept anything I suggest, or not. But please assume I have some reason for suggesting changes. If you did not want Scikit-HEP maintainers trying to help maintain iMinuit, you shouldn't have moved it to Scikit-HEP.